[18.0] [MIG] commission: Migration to 18.0

moaadbourhim opened the pull request.

AungKoKoLin1997 committed:
[MIG] sale_commission: Migration to 15.0

pedrobaeza committed:
[REF] *commission*: Abstract even more the base module

- Make specific top-level menu and permissions for commissions.
- Don't depend on account.
- Reorganize elements.
- Change some parts of the code to fit better ORM tools.
- Adjust permissions in children modules.
- Add <sheet> to some form views to get modern view.

pedrobaeza committed:
[IMP] commission*: Add settlement_type to commissions

This way, we can restrict which commissions we can select or be
automatically populate in the different documents.

Commit:
[UPD] Update commission.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
commission 15.0.2.0.0

Ryoko04 committed:
Added translation using Weblate (Japanese)

Ryoko04 committed:
Translated using Weblate (Japanese)

Currently translated at 53.7% (57 of 106 strings)

Translation: commission-15.0/commission-15.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-15-0/commission-15-0-commission/ja/

Ryoko04 committed:
Translated using Weblate (Japanese)

Currently translated at 61.3% (65 of 106 strings)

Translation: commission-15.0/commission-15.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-15-0/commission-15-0-commission/ja/

Ryoko04 committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (106 of 106 strings)

Translation: commission-15.0/commission-15.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-15-0/commission-15-0-commission/ja/

Commit:
[MIG] commission: Migration to 16.0

Commit:
[IMP] commission: Align dates in settlement view

Commit:
[UPD] Update commission.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

Commit:
[FIX] commission: Show full settlements table in partner

OCA-git-bot committed:
commission 16.0.1.1.0

alexeirivera87 committed:
[IMP] account_commission: multi-currency support

Commit:
[IMP] commission: Default commission_free instead of related

OCA-git-bot committed:
commission 16.0.1.1.1

jguenat committed:
[IMP] commission: use mail thread and improve report

Commit:
[UPD] Update commission.pot

OCA-git-bot committed:
commission 16.0.1.2.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

jguenat committed:
[FIXUP] improvements based on review

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (137 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/es/

dcorio committed:
Added translation using Weblate (Italian)

dcorio committed:
Translated using Weblate (Italian)

Currently translated at 81.0% (111 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 81.0% (111 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 81.0% (111 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 81.0% (111 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 86.1% (118 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 86.1% (118 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

dcorio committed:
Translated using Weblate (Italian)

Currently translated at 86.8% (119 of 137 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (136 of 136 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/pt_BR/

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 62.5% (85 of 136 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 98.5% (134 of 136 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/pt/

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 86.7% (118 of 136 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 86.7% (118 of 136 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

renda-dev committed:
[IMP] commission: Clarify selection fields

Commit:
[UPD] Update commission.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (138 of 138 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 84.0% (116 of 138 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/it/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (138 of 138 strings)

Translation: commission-16.0/commission-16.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-16-0/commission-16-0-commission/pt_BR/

ambientWave committed:
[MIG] commission: Migration to 17.0

ambientWave committed:
[IMP] commission: pre-commit auto fixes

Commit:
[UPD] Update commission.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (135 of 135 strings)

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/it/

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/

luis-ron committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (137 of 137 strings)

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/es/

Tisho99 committed:
[FIX] commission: partner view agent

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 94.8% (130 of 137 strings)

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 94.8% (130 of 137 strings)

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (137 of 137 strings)

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/pt/

SottomaiorMacedoTec committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (137 of 137 strings)

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/pt_BR/

Commit:
[UPD] Update commission.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/

metingulsoy committed:
Added translation using Weblate (Turkish)

metingulsoy committed:
Translated using Weblate (Turkish)

Currently translated at 94.1% (128 of 136 strings)

Translation: commission-17.0/commission-17.0-commission
Translate-URL: https://translation.odoo-community.org/projects/commission-17-0/commission-17-0-commission/tr/

moaadbourhim commented:
/ocabot migration commission

pedrobaeza commented:
Hi @moaadbourhim there are changes that shouldn't belong to `[IMP] commission: pre-commit auto fixes`. In fact, there shouldn't be any, as pre-commit shouldn't trigger anything before changing the version (and you mustn't change the manifest version yet). There are also some indentation changes that aren't correct.

/ocabot migration commission

Once corrected, you have to find reviewers:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#review

You can review other PRs, and ask in exchange that they review yours.

mb-andema committed:
[IMP] commission: pre-commit auto fixes

mb-andema committed:
[MIG] commission: Migration to 18.0

moaadbourhim commented:
Hi @pedrobaeza 

I have reviewed the link you shared and addressed the points mentioned. For the indentation issues, I used pre-commit to fix all the indentation problems.

Please let me know if anything else needs to be adjusted.

Thanks!

pedrobaeza commented:
It seems there's still the same issues I commented before.
